modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/display/context/DLMimeTypeDisplayContextUtil.java

```diff
@@ -15,43 +15,34 @@
 package com.liferay.wiki.web.internal.display.context;
 
 import com.liferay.document.library.display.context.DLMimeTypeDisplayContext;
+import com.liferay.osgi.util.service.Snapshot;
 import com.liferay.portal.kernel.repository.model.FileVersion;
 
-import org.osgi.framework.BundleContext;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
-import org.osgi.service.component.annotations.Reference;
-
 /**
  * @author Eudaldo Alonso
  */
-@Component(service = {})
 public class DLMimeTypeDisplayContextUtil {
 
 	public static String getCssClassFileMimeType(FileVersion fileVersion) {
-		return _dlMimeTypeDisplayContextUtil._dlMimeTypeDisplayContext.
-			getCssClassFileMimeType(fileVersion.getMimeType());
-	}
+		DLMimeTypeDisplayContext dlMimeTypeDisplayContext =
+			DLMimeTypeDisplayContextUtil._dlMimeTypeDisplayContextSnapshot.
```

Comment:
@MrJohn1911 No need to write `DLMimeTypeDisplayContextUtil._dlMimeTypeDisplayContextSnapshot` here.

Also, the PR title says layout and kb modules, but I see layout and wiki. Did you forget `com.liferay.knowledge.base.web.internal.display.context.DLMimeTypeDisplayContextUtil`?

Reply:
Oh, I picked the wrong class, I will resend with the right one. the `modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/frontend/js/loader/modules/extender/npm/NPMResolverProvider.java` was sent [here](https://github.com/liferay-core-infra/liferay-portal/pull/4349/files#diff-639a894ae0ba4825033dc5ad1e115c792f8b6a7cce462d0370f3e82daf280d88).